app/views/content/landing/still-time-to-apply.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Start your teacher training this September"
+description: There's still time for you to apply and start teacher training this September. 
+content:
+    - content/landing/still-time-to-apply/header
+    - content/landing/still-time-to-apply/content
+    - content/train-to-be-a-teacher/promos/adviser-promo-degree
+image: "static/images/content/hero-images/0027.jpg"
```

Comment:
@jenhadfield-dfe Would maybe suggest using one of the brand images rather than the TV image as they are stronger?

Otherwise page looks great!